openidm-zip/src/main/resources/samples/README.md

```diff
@@ -1,11 +1,50 @@
-OpenIDM Sample Projects and Sample Resources
-====================================
-Copyright (c) 2012-2016 ForgeRock AS
-This work is licensed under a Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License.
-See http://creativecommons.org/licenses/by-nc-nd/3.0/
+# What happened to samples?
 
-This directory contains samples to get you started with the many different configuration and security files you may
-need for your specific use case.
+Samples have been removed from the project because of their CC-BY-NC-ND license.
 
-The documentation at https://forgerock.org/openidm/doc/bootstrap/samples-guide/#chap-overview
-describes the purpose of each sample.
\ No newline at end of file
+We plan to create our own from the scratch, but that will take some time.
+
+# What can I do to test the functionality?
+
+Checkout this [commit](https://github.com/WrenSecurity/wrenidm/commit/ceb9579c81017b663a86ab6124a021bddb580251), you can still use the old samples from there.
```

Comment:
Please use link to a directory with samples (e.g. in sustaining branch), not commit.

Reply:
I have changed the link to point to `samples` directory in sustaining branch